packages/core/src/admin-ui/system/generateAdminUI.ts

```diff
@@ -63,33 +59,14 @@ export async function writeAdminFile (file: AdminFileToWrite, projectAdminPath:
   return Path.normalize(outputFilename)
 }
 
-const pageExtensions = new Set(['.js', '.jsx', '.ts', '.tsx'])
-
 export async function generateAdminUI (
   config: __ResolvedKeystoneConfig,
   graphQLSchema: GraphQLSchema,
   adminMeta: AdminMetaRootVal,
   projectAdminPath: string,
+  hasSrc: boolean,
   isLiveReload: boolean
 ) {
-  // when we're not doing a live reload, we want to clear everything out except the .next directory (not the .next directory because it has caches)
-  // so that at least every so often, we'll clear out anything that the deleting we do during live reloads doesn't (should just be directories)
-  if (!isLiveReload) {
```

Comment:
This part of live reload is not needed, we do not even need to figure out file linking from main dir to isolated dir anymore